src/java/org/apache/cassandra/db/virtual/AccordDebugKeyspace.java

```diff
@@ -297,12 +293,11 @@ private MaxConflictsTable()
                         "CREATE TABLE %s (\n" +
                         "  keyspace_name text,\n" +
                         "  table_name text,\n" +
-                        "  token_sort blob,\n" +
-                        "  token_start text,\n" +
-                        "  token_end text,\n" +
                         "  command_store_id bigint,\n" +
+                        "  token_start 'TokenUtf8Type',\n" +
```

Comment:
Nit: is it possible to give it an alias? like token_string or something similar?

Reply:
I guess we could hard code special casing in the type parser? Or introduce an internal alias mechanism for use there, sure